src/main/java/com/algosenpai/app/logic/command/HelpCommand.java

```diff
@@ -23,18 +24,47 @@ public String execute() {
             return getDefaultMessage();
         } else {
             int index = userStats.getIndexByName(inputs.get(1));
+            userStats = userStats.parseString(Storage.loadData("UserData.txt"));
```

Comment:
https://nuscs2113-ay1920s1.github.io/website/schedule/week6/topics.html
This code seems to have the arrowhead style code mentioned during lecture. Consider breaking the method down.

Reply:
Agreed. @nwjbrandon raises an excellent point here and it cannot be ignored.